.github/release.yml

```diff
@@ -10,6 +10,10 @@ changelog:
     - title: Changes
       labels:
         - "*"
+      exclude:
+        labels:
+          - bug
+          - meta
```

Comment:
Doesn't `enhancement` need to be excluded here, too?

Reply:
No, it should be "captured" by the "New" category before (cf. the [example config](https://docs.github.com/en/repositories/releasing-projects-on-github/automatically-generated-release-notes#example-configuration)).

Reply:
Ah, so `bug` is not "captured" because the "Bugfixes" category comes after the "Changes" category?

And `meta` needs to be repeated because a category's `excludes` overrides the global `excludes`?

Reply:
`meta` is here because we don't want to include such changes

Reply:
Fixed in https://github.com/refined-github/refined-github/pull/5522, thanks for the comment